### ISSUE TITLE
[Feature Request] Custom page URLs

### ISSUE THREAD

**Issue by modem7** [vladkens/ghstats#8]
This is probably more a personal thing rather than anything else, but the ability to add 1-3 custom URLs (with titles) would be a nice to have feature. 

E.g.  At the top or bottom of the page (maybe configurable?), I could have: 1. My personal website, 2. My linkedin, 3. My wiki.

Reason I've recommended 1-3 is that it wouldn't clutter things up. 

Potential env var: 

```
CUSTOM_URL=mywiki|https://www.modem7.com,linkedin|https://linkedin.com/in/myuser,someothersite|www.myothersite.com
CUSTOM_URL_POS=top
```

E.g.
![Untitled](https://github.com/user-attachments/assets/9cd3f701-b369-49d1-a920-8eeb825d3468)

**Comment by vladkens** [comment]
Hi, @modem7.

Will be added in next release:
<img width="1422" alt="image" src="https://github.com/user-attachments/assets/0f55c0f9-41fe-42ea-98eb-53b3ef1e1247">

**Comment by vladkens** [comment]
@modem7 released in [v0.3.0](https://github.com/vladkens/ghstats/releases/tag/v0.3.0)
